src/main/java/com/ericsson/ei/rules/RulesHandler.java

```diff
@@ -39,38 +44,91 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 @Component
-@Scope(value="thread", proxyMode = ScopedProxyMode.TARGET_CLASS)
+@Scope(value = "thread", proxyMode = ScopedProxyMode.TARGET_CLASS)
 public class RulesHandler {
     private static final Logger LOGGER = LoggerFactory.getLogger(RulesHandler.class);
 
-    @Value("${rules.path}") private String jsonFilePath;
+    private static final String EI_HOME_DEFAULT_NAME = ".eiffel-intelligence";
+
+    @Value("${rules.path}")
+    private String jsonFilePath;
```

Comment:
jsonFilePath could maybe be switched out to another more explaining name maybe (rulesFilePath for example)

Reply:
Done.